Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.6-alpine
+
+ENV HOME /home
+RUN mkdir ~/.aws && mkdir ~/app
+
+## CONFIGURE AWS
+COPY config /home/.aws/
```

Comment:
where is `config` defined? Can't seem to find it in this PR

Reply:
I thought it was needed by boto3 but on further testing, it looks like it is not needed.